### PR TITLE
feat: add Slipstream volume + fees

### PR DESCRIPTION
**Pull request by ethzoomer**
Based the code off of `dexs/aerodrome`. Verified the outputted numbers with Velodrome Sugar repo results and Velodrome UI

### CODE REVIEW

**Comment by llamatester** [comment]
The velodrome-slipstream adapter exports: 
        
 
 _______________________________________
    Dexs for 21/4/2024
    _______________________________________
    
    OPTIMISM 👇
    Backfill start time: 6/3/2024
    NO METHODOLOGY SPECIFIED
    Daily volume: 30.35 M
    Timestamp: 1713743999 (2024-04-21T23:59:59.000Z)
    
    
    
    
    Done in 16.59s.

**Comment by llamatester** [comment]
The velodrome-slipstream adapter exports: 
        
 
 _______________________________________
    Fees for 21/4/2024
    _______________________________________
    
    OPTIMISM 👇
    Backfill start time: 6/3/2024
    NO METHODOLOGY SPECIFIED
    Daily fees: 14.43 k
    Timestamp: 1713743999 (2024-04-21T23:59:59.000Z)
    Daily revenue: 14.43 k
    Daily holders revenue: 14.43 k
    
    
    
    
    Done in 7.22s.

File: dexs/velodrome-slipstream/index.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  const logs: ILog[][] = await getLogs({
+    targets,
+    eventAbi: event_swap,
+    flatten: false,
+  })
+
+  logs.forEach((logs: ILog[], idx: number) => {
+    const { token0, token1 } = forSwaps[idx]
+    logs.forEach((log: any) => {
+      dailyVolume.add(token0, BigInt(Math.abs(Number(log.amount0))))
```
Comment (0xngmi) [inline dexs/velodrome-slipstream/index.ts]: doesnt this doublecount volume?

Comment (dtmkeng) [inline dexs/velodrome-slipstream/index.ts:49]: this line is double count volume it should only one `amount1` or `amount0`

**Review by dtmkeng — CHANGES_REQUESTED** [review]

**Comment by llamatester** [comment]
The velodrome-slipstream adapter exports: 
        
 
 _______________________________________
    Dexs for 22/4/2024
    _______________________________________
    
    OPTIMISM 👇
    Backfill start time: 6/3/2024
    NO METHODOLOGY SPECIFIED
    Daily volume: 15.94 M
    Timestamp: 1713830399 (2024-04-22T23:59:59.000Z)
    
    
    
    
    Done in 18.35s.

**Comment by llamatester** [comment]
The velodrome-slipstream adapter exports: 
        
 
 _______________________________________
    Fees for 22/4/2024
    _______________________________________
    
    OPTIMISM 👇
    Backfill start time: 6/3/2024
    NO METHODOLOGY SPECIFIED
    Daily fees: 10.88 k
    Timestamp: 1713830399 (2024-04-22T23:59:59.000Z)
    Daily revenue: 10.88 k
    Daily holders revenue: 10.88 k
    
    
    
    
    Done in 7.02s.

File: dexs/velodrome-slipstream/index.ts (hunk 1 above)
Comment (ethzoomer) [reply dexs/velodrome-slipstream/index.ts]: Yes ty fixed

**Pull request merged**